web/src/app/projects/view/view.component.ts

```diff
@@ -91,6 +94,31 @@ export class ViewProjectComponent implements OnInit, OnDestroy {
       .subscribe();
   }
 
+  /**
+   * Increase the followers count by 1 and update the followings in the users collection
+   */
+  incrementFollowers(): void {
+    this.projectService.updateFollowers(this.project.uid, 1).subscribe();
+    this.userService.updateFollowings(this.authService.profile.uid, this.project.uid, true).
```

Comment:
The `profile.uid` could be used directly in the `userService` would not need to be included in and then passed through

Reply:
ok